packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 46.11.0 - August 22, 2019
+
+## Fixed
+
+#### salesforcedx-core
+
+- Add CustomObjects as a bundle type for retrieval with the Org Browser ([PR #1569](https://github.com/forcedotcom/salesforcedx-vscode/pull/1569))
+
+- Fix retrieval of standard objects for the Org Browser ([PR #1571](https://github.com/forcedotcom/salesforcedx-vscode/pull/1571))
+
+#### docs
+
+- Fix Org Browser documentation link ([PR #1565](https://github.com/forcedotcom/salesforcedx-vscode/pull/1565))
+
+## Added
+
+#### salesforcedx-core
+
+- Open beta for Diff Plugin. Allow diffing for local metadata against an org ([PR #1568](https://github.com/forcedotcom/salesforcedx-vscode/pull/1568))
```

Comment:
I'm not sure if we want to include this for now. We might need some additional information first so people know how to activate and use this feature.